torch_xla/core/dynamo_bridge.py

```diff
@@ -417,11 +417,19 @@ def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
 
   # partition the model and exectue to collect inputs
   supported_ops = XlaOperatorSupport()
-  partitioner = CapabilityBasedPartitioner(xla_model, supported_ops)
+  partitioner = CapabilityBasedPartitioner(
+      xla_model, supported_ops, allows_single_node_partition=True)
   partitions = partitioner.propose_partitions()
   partitioned_graph = partitioner.fuse_partitions(partitions)
   InputCollector(partitioned_graph).run(*xla_args)
 
+  # TODO (@wonjoo) Add some comment
+  self_tensors = []
+  for name, buffer in xla_model.named_buffers():
```

Comment:
what else can be in the `named_buffers`? beside the `self.xxx`

Reply:
For any of our other tests in dynamo, there is nothing in `named_buffers`. In the newly added test cases where the model has `self.tensor`, the only entry in named_buffers is its `self.tensor`:
```
[WONJOO] name=L__self___self_tensor
[WONJOO] buffer=tensor([[1., 2., 3.],
        [0., 0., 0.],
        [7., 8., 9.],
        [0., 0., 0.],
        [4., 5., 6.]], device='xla:0')
```